src/commonMain/kotlin/exchange.dydx.abacus/state/manager/StateManagerAdaptor.kt

```diff
@@ -1802,15 +1802,11 @@ open class StateManagerAdaptor(
         val reduceOnly = true
         val postOnly = false
 
-        val timeInForce = when (triggerOrder.type) {
-            OrderType.stopMarket, OrderType.takeProfitMarket -> "IOC"
-            OrderType.stopLimit, OrderType.takeProfitLimit -> "GTT"
-            else -> throw Exception("invalid triggerOrderType")
-        }
+        val timeInForce = null;
 
         val execution = when (triggerOrder.type) {
             OrderType.stopMarket, OrderType.takeProfitMarket -> "IOC"
```

Comment:
this could also be FOK or IOC here. I wonder if this should be set to DEFAULT as well? Unsure if that's supported for non-limit TP/SL orders

Reply:
ah so product confirmed that we want all non-limit TP/SL orders to have execution "IOC" (users can't change this)